db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :playbook_name, :string
```

Comment:
@Fryguy any change of opinion on the serialized column based on https://github.com/ManageIQ/manageiq/pull/13855#discussion_r100433630 ?